src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -65,9 +67,63 @@ public AgentSettings LoadSettings()
             return settings;
         }
 
+        private void checkAgentRootDirectorySecure()
```

Comment:
This method should be capitalized and placed below the public ones.

Reply:
Thanks for the point! Fixed it!